workflow-manager/main.go

```diff
@@ -451,19 +418,17 @@ func scheduleTasks(config scheduleTasksConfig) error {
 		Int("incomplete peer validations", peerValidationBatches.IncompleteBatchCount).
 		Msg("discovered peer validations")
 
-	// Take the intersection of the sets of own validations and peer validations
+	// Take the intersection of the sets of ingestion batches and peer validations
 	// to get the list of batches we can aggregate.
-	// Go doesn't have sets, so we have to use a map[string]bool. We use the
-	// batch ID as the key to the set, because batchPath is not a valid map key
-	// type, and using a *batchPath wouldn't give us the lookup semantics we
-	// want.
-	ownValidationsSet := map[string]bool{}
-	for _, ownValidationBatch := range ownValidationBatches.Batches {
-		ownValidationsSet[ownValidationBatch.ID] = true
+	// Go doesn't have sets, so we have to use a map[string]struct{}, with the batch ID being used
+	// as the map key.
```

Comment:
nit: The Go set idiom is a pet peeve of mine so I have a habit of calling it out, but I think it's familiar enough to Go programmers that we can omit the explanatory comment.